packages/conform-react/context.tsx

```diff
@@ -272,24 +260,6 @@ export function getMetadata<
 
 				return result;
 			},
-			get getFieldset() {
-				return () =>
-					new Proxy({} as any, {
-						get(target, key, receiver) {
-							if (typeof key === 'string') {
-								return getFieldMetadata(
-									context,
-									subjectRef,
-									stateSnapshot,
-									name,
-									key,
-								);
-							}
-
-							return Reflect.get(target, key, receiver);
-						},
-					});
-			},
```

Comment:
Why?

Reply:
@edmundhung based off the type it doesn't seem to be used, is it? tests still seem to pass.